src/loaders/lottie/tvgLottieModel.h

```diff
@@ -590,6 +592,25 @@ struct LottieLayer : LottieGroup
 };
 
 
+struct LottieSlot
+{
+    char* sid = nullptr;
+
+    enum Type : uint8_t
```

Comment:
Could you please cherry pick this first, and make this based on it?
https://github.com/thorvg/thorvg/pull/1998